chart/identity-manager-sts/values.yaml

```diff
@@ -0,0 +1,98 @@
+---
+image:
+  repository: ""
+  pullPolicy: IfNotPresent
+  tag: ""
```

Comment:
We could use `ghcr.io/infratographer/identity-manager-sts:latest` here if we want, since that's already being pushed up today.